js/app.js

```diff
@@ -71,8 +71,15 @@ function printReport() {
     data: {
       labels: getNames(),
       datasets: [{
-        label: 'Popularity',
-        data: getPopularity(),
+        label: 'Times Clicked',
+        data: getPopularity()[0],
```

Comment:
maybe a bit clearer to be able to say 'getPopularity()['numClicks']' by returning ' {'numClicks': numClicks, 'numShows': numShows}'